.claude/commands/pm/epic-start.md

```
@@ -0,0 +1,247 @@
+---
+allowed-tools: Bash, Read, Write, LS, Task
+---
+
+# Epic Start
+
+Launch parallel agents to work on epic tasks in a shared branch.
+
+## Usage
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)